Refresh.GameServer/Middlewares/DigestMiddleware.cs

```diff
@@ -1,114 +1,158 @@
-using System.Diagnostics;
 using System.Security.Cryptography;
+using System.Text;
 using Bunkum.Listener.Request;
 using Bunkum.Core.Database;
 using Bunkum.Core.Endpoints.Middlewares;
 using Refresh.Common.Extensions;
+using Refresh.GameServer.Authentication;
+using Refresh.GameServer.Configuration;
+using Refresh.GameServer.Database;
 using Refresh.GameServer.Endpoints;
 
 namespace Refresh.GameServer.Middlewares;
 
-public class DigestMiddleware : IMiddleware
+public class DigestMiddleware(GameServerConfig config) : IMiddleware
```

Comment:
I'd prefer to avoid primary constructors

Reply:
removed in latest commit